src/fileListUnzip.ts

```diff
@@ -0,0 +1,24 @@
+import { fileListFromZip } from './fileListFromZip';
+
+interface FileList {
```

Comment:
this type not a file list (contains data about 1 file)

Reply:
Also, I think we should not redefine FileList. It's ambiguous with the builtin from browsers

Reply:
Yes this was in my comment to reuse PartialFileList

export type PartialFile = Omit<File, 'stream' | 'slice' | 'type'>;
export type PartialFileList = PartialFile[];